google/cloud/security/inventory/inventory_loader.py

```diff
@@ -12,27 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Loads data into Inventory.
-
-Usage:
```

Comment:
What happens when someone runs

forseti_inventory

Does the usage example with the flags still show in the CLI?